website/backend/press/models.py

```diff
@@ -0,0 +1,19 @@
+from django.db import models
+from backend.utils.models import BaseModel
+from author.decorators import with_author
+from cloudinary.models import CloudinaryField
+
+@with_author
+class Press(BaseModel):
+    article_title = models.CharField(max_length=100)
+    article_intro = models.CharField(max_length=200, null=True, blank=True)
+    article_link = models.URLField(null=True, blank=True)
+    date_published = models.DateField()
+    publisher_logo = CloudinaryField("PublisherLogo", overwrite=True, resource_type="image", null=True, blank=True)
+    order = models.IntegerField(default=1)
+
```

Comment:
@OchiengPaul442 Let me know if you'll need any other attributes on the frontend

Reply:
I will. thanks, @FaithDaka

Reply:
Hi @FaithDaka , just a followup on the Cloudinary file orgnaisation. Are we storing these files in a "nice way"? Say like in some hierarchical folder structure of sorts for easy management?